domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/stops/LocationBase.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.stops;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.mobilitydata.gtfsvalidator.domain.entity.gtfs.GtfsEntity;
+import org.mobilitydata.gtfsvalidator.domain.entity.notice.base.Notice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Base class for all entities defined in stops.txt
+ */
+public abstract class LocationBase extends GtfsEntity {
+
+    private @NotNull
+    final String stopId;
+    private final String stopCode;
+    private final String stopName;
+    private final String stopDesc;
```

Comment:
It's official, it should be added to this PR.

Reply:
Does the extension include other changes? I would be inclined to fully implement in a specific PR and not do partial implementation if that's the case. Otherwise yes, that should be added.

Reply:
No, it's just the single field. I think when you started this PR for some reason the `tts_stop_name` field wasn't showing up on gtfs.org at the time (auto-sync with google/transit broke?), which is probably how it was missed.

Reply:
Also, our current goal is to attain feature parity with the pyhon validator.